tests/e2e/calico/calico.go

```diff
@@ -0,0 +1,293 @@
+package calico
```

Comment:
Why do we need this code for these tests? Can we add a note about why we're vendoring it, and talk to cloud platform about extracting it to a separate repo if we need to use it in multiple repos?